tick/plot/plot_stem.py

```diff
@@ -85,3 +92,116 @@ def stems(ys: list, titles: list = None, show=True, sync_axes=True):
         return None
     else:
         return p
+
+
+def stems_matplotlib(ys: list, titles: list = None, show=True, sync_axes=True):
+    """Several stem plots with synchronized axes using bokeh rendering
+
+    Parameters
+    ----------
+    ys : `list` of `np.ndarray`
+        A list of numpy arrays to be plotted
+
+    titles : `list` of `str`
+        The titles of each plot
+
+    show : `bool`, default=`True`
+        if True, show the plot. Use False if you want to superpose several
+        plots
+
+    sync_axes : `bool`, default=`True`
+        If True, the axes of the stem plot are synchronized
+
+    Returns
+    -------
+    output : `bokeh.models.layouts.Column` or `None`
+        A bokeh object containing the layout of the plot, when ``show=False``,
+        None otherwise.
+    """
+    import matplotlib.pyplot as plt
+    figs = []
+    x_range = None
+    y_range = None
+    plt.figure(figsize=())
+    for idx, y in enumerate(ys):
+        if titles is not None:
+            title = titles[idx]
+        else:
+            title = None
+        fig = plt.stem(y, title=title, x_range=x_range, y_range=y_range)
+        figs.append(fig)
+        if idx == 0 and sync_axes:
+            x_range = fig.x_range
+            y_range = fig.y_range
+    p = bk.gridplot([[e] for e in figs])
+    if show:
+        bk.show(p)
+        return None
+    else:
+        return p
+
+
+def stems(ys: list, titles: list = None, show=True, sync_axes=True,
+          rendering: str='bokeh'):
```

Comment:
Don't we use matpllotlib by default anywhere? Bokeh is not a requirement of tick